### DISCUSSION TITLE
Does this plugin support having multiple versions of a LS installed?

### DISCUSSION THREAD

**Discussion by ghost** [mason-org/mason.nvim#1409] [Q&A] [answered]
One of my requirements is to have different LS versions per project.
Is it possible to accomplish something like that with this plugin?

Thank you

**Comment by williamboman** [comment, marked as answer]
Hello! No not yet. What you could do is use a different `install_root_dir` when setting up Mason. This will however have the drawback of only being able to "activate" a single project's installed packages in each Neovim instance, i.e. you can only reliably set up Mason once.

```lua
local expand = vim.fn.expand
local root_dir_overrides = {
  [expand "~/dev/project-A"] = expand "~/mason-project-A",
  [expand "~/dev/project-B"] = expand "~/mason-project-B",
}

require("mason").setup {
  install_root_dir = root_dir_overrides[vim.loop.cwd()]
}
```
